README.md

```diff
@@ -67,10 +67,10 @@ response in a text view.
 You can build the application by running:
 
 ```
-$ bazel build //ios-app:GetUrl
+$ bazel build ios-app --ios_sdk_version=8.4
```

Comment:
We should know 1) will this break people with older/newer versions of XCode? 2) Does this need to be set to something different for other versions of xcode?

Minor thing: just to be consistent with targets above, please make it //ios-app.